powerline/bindings/wm/__init__.py

```diff
@@ -0,0 +1,31 @@
+# vim:fileencoding=utf-8:noet
```

Comment:
I asked for i3 and xrandr in one module because it is `….bindings.wm` and both are wm-related. If there were more functions like this it would make sense to use `….wm` and `….i3`, but currently as i3 ∈ wm they may be in the same file.